src/app/views/teams/teams-view/TeamsList.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router";
+import url from "../../../utilities/url";
+import Input from "../../../components/Input";
+import SimpleSelectableList from "../../../components/simple-selectable-list/SimpleSelectableList";
+
+const propTypes = {
+    dispatch: PropTypes.func,
+    teams: PropTypes.object.isRequired,
+    loadTeams: PropTypes.func.isRequired,
+};
+
+const TeamsList = props => {
+    const { teams, loadTeams } = props;
+
+    const [isFetchingTeams, setIsFetchingTeams] = useState(true);
+    const [filterTerm, setFilterTerm] = useState("");
+    const [previewTeams, setPreviewTeams] = useState([]);
+
+    useEffect(() => {
+        loadTeams();
+    }, []);
+
+    useEffect(() => {
```

Comment:
can you please utilise the work around the fetching groups is already implements here: https://github.com/ONSdigital/florence/blob/develop/src/app/config/thunks.js#L146
and used in: https://github.com/ONSdigital/florence/blob/develop/src/app/views/users/groups/AddGroupsToUser.jsx

Reply:
you never set isFetchingTeams to true so the loading state is not visible